code/datums/uplink_item.dm

```diff
@@ -942,6 +942,14 @@ var/list/uplink_items = list()
 	gamemodes = list(/datum/game_mode/nuclear)
 	excludefrom = list(/datum/game_mode/gang)
 
+/datum/uplink_item/suits/hardsuit/shielded
+	name = "Shielded Hardsuit"
+	desc = "An advanced hardsuit with built in energy shielding. The shields will rapidly recharge when not under fire."
+	item = /obj/item/clothing/suit/space/hardsuit/shielded/syndi
+	cost = 30
+	gamemodes = list(/datum/game_mode/nuclear)
+	excludefrom = list(/datum/game_mode/gang)
```

Comment:
remove this shit,it's not used, if you see it up here it's because it's a leftover of an idea, gang gamemode doesn't use uplink items